server/controllers/question.controller.js

```diff
@@ -15,6 +18,41 @@ async function get(req, res) {
   return res.json(question.toObject());
 }
 
+async function getUnanswered(req, res) {
+  let questions = await Question
+    .find({
+      answers: { $size: 0 },
+      $or: [
+        { deleted: false },
+        { deleted: { $exists: false } },
+      ],
+    })
+    .populate('author', 'name lastName email website avatarUrl bio');
+
+  if (!questions.length) {
+    return res.status(404).send('Not found');
```

Comment:
Sure that this is a return error and not just empty array?

Reply:
Good point!